vitrina/resources/models.py

```diff
@@ -40,70 +43,125 @@ class DatasetDistribution(models.Model):
     # TODO: https://github.com/atviriduomenys/katalogas/issues/59
     created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     modified = models.DateTimeField(blank=True, null=True, auto_now=True)
-    version = models.IntegerField()
+    version = models.IntegerField(default=1)
     deleted = models.BooleanField(blank=True, null=True)
     deleted_on = models.DateTimeField(blank=True, null=True)
 
-    identifier = models.CharField(max_length=255, blank=True, null=True)
+    dataset = models.ForeignKey(Dataset, models.CASCADE)
 
-    title = models.TextField(blank=True, null=True)
-    description = models.TextField(blank=True, null=True)
+    identifier = models.CharField(max_length=255, blank=True, null=True)
 
-    period_end = models.CharField(max_length=255, blank=True, null=True)
-    period_start = models.CharField(max_length=255, blank=True, null=True)
+    title = models.CharField(
+        blank=True,
+        max_length=255,
+        verbose_name=_('Pavadinimas'),
+    )
+    description = models.TextField(
+        blank=True,
+        verbose_name=_('Aprašymas'),
+    )
+
+    access_url = models.CharField(
+        max_length=255,
+        blank=True,
+        verbose_name=_('Prieigos nuoroda'),
+        help_text=_(
+            'Nuoroda į svetainę iš kurios galima atsisiųsti duomenis.'
+        ),
+    )
+
+    format = models.ForeignKey(
+        Format,
+        models.SET_NULL,
+        blank=False,
+        null=True,
+        verbose_name=_('Duomenų formatas'),
+    )
+
+    download_url = models.TextField(
+        blank=True,
+        null=True,
+        verbose_name=_('Atsisiuntimo nuoroda'),
+        help_text=_(
+            'Tiesioginė duomenų atsisiuntimo nuoroda.'
+        ),
+    )
+
+    file = models.FileField(
+        upload_to='data/',
+        max_length=255,
+        blank=True,
+        null=True,
+        verbose_name=_('Duomenų failas'),
+        help_text=_(
+            'Atvirų duomenų katalogas nėra skirtas duomenų talpinimui ir '
+            'įprastinių atveju duomenys turėtu būti talpinami atvirų duomenų '
+            'Saugykloje ar kitoje vietoje, pateikiant tiesioginę duomenų '
+            'atsisiuntimo nuorodą. Tačiau nedidelės apimties (iki 5Mb) '
+            'duomenų failus, galima talpinti ir kataloge.'
+        ),
+    )
+
+    geo_location = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
```

Comment:
```suggestion
```

Char fields should never be null, unless there is a good reason for that. Please change this also for `period_start` and `period_end`.

Also, when adding a form with empty `geo_location`, `period_start` or `period_end` I get this error:

```python
Traceback (most recent call last):
  File "django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "geo_location" of relation "dataset_distribution" violates not-null constraint
DETAIL:  Failing row contains (10938, 2022-10-10 18:15:26.796594+00, 2022-10-10 18:15:26.796629+00, 1, , null, , 
null, null, null, null, null, null, , URL, http://example.com/data.csv, 1254, null, null, null, 1, , null).
```

It looks, that model says null=True, but migrations things it's null=False. Anyway, CharField, should almost never be null=True. Empty string fully satisfies emptiness, there is no need for special kind of emptiness for strings.